_pages/it/newst.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Out of curiosity: what does this file do? And why the `newst` name?

Reply:
was a safe renaming of old news pages structure. Now they no longer exist.

Reply:
No, we do not need it. Deleted.